docs/modules/plugins/pages/plugin-mobile-app.adoc

```diff
@@ -696,3 +696,28 @@ When I type text `pas1$`
 ----
 
 include::partial$generic-ui-steps.adoc[]
+
+=== Recording iOS Performance metrics
+
+Records performance metrics of iOS application. It is based on the Instruments utility distributed by Xcode which provides a number of built-in analyses and measurements.
+See https://appiumpro.com/editions/12-capturing-performance-data-for-native-ios-apps[Capturing Performance Data for Native iOS Apps] and https://developer.apple.com/videos/play/wwdc2019/411/[Getting Started with Instruments] for more information.
+
+[NOTE]
+Appium server should be started with the *--relaxed-security* flag. This is because Instruments can gather data from the system as a whole, not just the AUT. (It's thus a security risk to expose potentially sensitive system information to Appium sessions running from a remote client).
+
+[source,gherkin]
+----
+When I start recording `$instrument` metrics
+When I stop recording `$instrument` metrics and save results to file `$path`
+----
+
+. `$instrument` - The instrument of Xcode Instruments utility
```

Comment:
I believe it was already mentioned in the previous comments, but not fixed yet: please enumerate the available instruments or provide the link how to find the list